.gitignore

```diff
@@ -354,3 +354,4 @@ src/Playwright.sln.DotSettings
 src/Playwright/Drivers
 src/Playwright/playwright
 src/Playwright.Tests/Screenshots/**/test.png
+src/Playwright/Microsoft.Playwright.xml
```

Comment:
Is this ok @avodovnik?

Reply:
Yeah, this should be fine! I suspect that's why it was set to be in the `bin` folder before?